vendor/raylib/raylib.odin

```diff
@@ -925,13 +925,17 @@ NPatchLayout :: enum c.int {
 	THREE_PATCH_HORIZONTAL,  // Npatch layout: 3x1 tiles
 }
 
-
+// NOTE: Castable to `core:c/libc`'s `va_list`.
+// But some use cases of raylib do not want `libc` imported.
+va_list :: struct #align(16) {
+	_: [4096]u8,
+}
```

Comment:
Being castable might be fine, but this seems off.

Reply:
What if we put the `va_list` definition in `core:c` and alias it in `core:c/libc`? That seems clean to me

Reply:
Might be a good idea.